src/app/forms/templates/psv/psv-dda.template.ts

```diff
@@ -13,73 +14,139 @@ export const PsvDdaTemplate: FormNode = {
           name: 'certificateIssued',
           label: 'DDA certificate issued',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
+          editType: FormNodeEditTypes.RADIO,
+          options: [
+            { value: true, label: 'Yes' },
+            { value: false, label: 'No' },
+            { value: '', label: 'I do not know' }
+          ],
         },
         {
           name: 'wheelchairCapacity',
           label: 'Wheelchair capacity',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
+          editType: FormNodeEditTypes.NUMBER,
+          validators: [
+            { name: ValidatorNames.Max, args: 99 }
+          ],
         },
         {
           name: 'wheelchairFittings',
           label: 'Wheelchair fittings',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
+          editType: FormNodeEditTypes.TEXTAREA,
+          validators: [
+            { name: ValidatorNames.MaxLength, args: 250 }
+          ],
         },
         {
           name: 'wheelchairLiftPresent',
           label: 'Wheelchair lift present',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
+          editType: FormNodeEditTypes.RADIO,
+          options: [
+            { value: true, label: 'Yes' },
+            { value: false, label: 'No' },
+            { value: '', label: 'I do not know' }
+          ],
         },
         {
           name: 'wheelchairLiftInformation',
           label: 'Wheelchair lift information',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
+          editType: FormNodeEditTypes.TEXTAREA,
+          validators: [
+            { name: ValidatorNames.MaxLength, args: 250 }
+          ],
         },
         {
           name: 'wheelchairRampPresent',
           label: 'Wheelchair ramp present',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
+          editType: FormNodeEditTypes.RADIO,
+          options: [
```

Comment:
Might be worth adding a validator to this:
`validators: [{ name: ValidatorNames.Required }]`